src/posts/paper/14-research-sources.ts

```diff
@@ -91,12 +93,18 @@ if height = 8ft,
 CFM = 2/3 * (square footage)
 \`\`\`
 
-From this, we also deduce that areas that have high rates of air exchange while not being "outside" per-se should be similarly safer. This includes cars with the windows rolled down and partially enclosed spaces (i.e. at least one wall is open to outside air).
+###### Moving car with windows rolled down; a space with one or more sides open to the outdoors
```

Comment:
Perhaps easier on the eyes / easier to skip if this is two sections separate sections?

Reply:
@beshaya I went ahead and separated them. Whatcha think?

Reply:
I like how it looks separated!